### PR TITLE
Add an email to teams about kit disclaimers & kickstart

### PR DESCRIPTION
**Pull request by PeterJCLaw**
closes https://github.com/srobo/tasks/issues/960

### CODE REVIEW

File: SR2023/2022-10-19-kickstart-and-kit-disclaimer.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+another team. If this is going to be unachievable please let us know and we can
+work something out.
+
+## Kickstart
+
+As you already know, [Kickstart][kickstart] is this Saturday 19th October. For
+those attending in person please arrive at 9:45am at [Building 46][B46] on
+Highfield Campus.
+
+The presentation, which will be livestreamed from there, will start at 10am UK
```
Comment (RealOrangeOne) [inline SR2023/2022-10-19-kickstart-and-kit-disclaimer.md:35]: Suggestion: We should explicitly say where the livestream link is

**Review by RealOrangeOne — APPROVED** [review]

File: SR2023/2022-10-19-kickstart-and-kit-disclaimer.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+As it's a legal document, please print out and sign the disclaimer form
+attached. If you're coming to Kickstart please bring it with you on the day and
+we'll also have spare copies printed out. Otherwise please scan it and email it
+back to us.
+
+For those not coming to Kickstart we will send out your kit to the address
+listed below. Please let us know explicitly if this address is correct, and if
+it's not, let us know an updated one.
+
+Address:
```
Comment (sedders123) [inline SR2023/2022-10-19-kickstart-and-kit-disclaimer.md:23]:
```suggestion
For those not coming to Kickstart we will send out your kit to you.
Please reply to this email to confirm which address we should send the kit to.
```

File: SR2023/2022-10-19-kickstart-and-kit-disclaimer.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2023/2022-10-19-kickstart-and-kit-disclaimer.md]: https://github.com/srobo/team-emails/pull/117/commits/25ea81e5e4ee7770ecd04a1efd82cbbc24240274

**Review by Adimote — APPROVED** [review]

File: SR2023/2022-10-19-kickstart-and-kit-disclaimer.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+As you already know, [Kickstart][kickstart] is this Saturday 19th October. For
+those attending in person please arrive at 9:45am at [Building 46][B46] on
+Highfield Campus.
+
+The presentation, which will be [livestreamed][livestream] from there, will
+start at 10am UK time and last for approximately an hour. This will explain this
+year’s game, the competition structure, and give an overview of the kit.
+
+The rest of the day will provide an opportunity for your teams to get started on
```
Comment (Adimote) [inline SR2023/2022-10-19-kickstart-and-kit-disclaimer.md:36]: No mention of how long this is

**Pull request merged**